client/blocks/import/style/base.scss

```diff
@@ -182,7 +182,8 @@
 		position: absolute;
 		top: 1px;
 		left: 0;
-		fill: var(--studio-gray-20);
+		color: var(--studio-gray-20);
+		fill: currentColor;
```

Comment:
Switching to `color` + `fill` for a more comprehensive way to change the logo's color that is backwards compatible, but also works with the new version of the jetpack logo